specreduce/tests/test_background.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+import astropy.units as u
+from astropy.nddata import CCDData
+
+from specreduce.background import Background
+from specreduce.tracing import FlatTrace
+
+
+# Test image is comprised of 30 rows with 10 columns each. Row content
+# is row index itself. This makes it easy to predict what should be the
+# value extracted from a region centered at any arbitrary Y position.
+image = np.ones(shape=(30, 10))
+for j in range(image.shape[0]):
+    image[j, ::] *= j
+image = CCDData(image, unit=u.Jy)
```

Comment:
A note here that this is the same test image as in `specreduce/tests/test_extract.py` might be helpful for ease of future reading.